sdk/containerregistry/container-registry/README.md

```diff
@@ -0,0 +1,80 @@
+# Azure Container Registry client library for JavaScript
+
+Azure [Container Registry](https://azure.microsoft.com/services/container-registry/) is a managed, private Docker registry service based on the open-source Docker Registry 2.0. Create and maintain Azure container registries to store and manage your private Docker container images and related artifacts.
+
+Use Azure container registries with your existing container development and deployment pipelines, or use Azure Container Registry Tasks to build container images in Azure. Build on demand, or fully automate builds with triggers such as source code commits and base image updates.
+
+## Getting started
+
+### Currently supported environments
+
+- Node.js version 8.x or higher
+
+### Prerequisites
+
+- An [Azure subscription][azure_sub].
+- An [Azure Container Registry resource][acr_resource]
+
+Usually you'd put a shell command for provisioning the necessary Azure services here.
+
+### Install the `@azure/container-registry` package
+
+Install the Container Registry client library for JavaScript with `npm`:
+
+```bash
+npm install @azure/container-registry
+```
+
+### Browser support
+
+#### JavaScript Bundle
+
+To use this client library in the browser, first you need to use a bundler. For details on how to do this, please refer to our [bundling documentation](https://aka.ms/AzureSDKBundling).
+
+### Further examples
+
+Top-level examples usually include things like creating and authenticating the main Client. If your service supports multiple means of authenticating (e.g. key-based and Azure Active Directory) you can give a separate example of each.
+
+## Key concepts
+
+### RepositoryClient
+
+`RepositoryClient` provides operations to interact with a repository.
+
+## Examples
+
+### First Example
+
+<!-- Examples should showcase the primary, or "champion" scenarios of the client SDK. -->
+
+Create several code examples for how someone would use your library to accomplish a common task with the service.
+
+## Troubleshooting
+
+### Enable logs
+
+You can set the following environment variable to see debug logs when using this library.
+
+- Getting debug logs from the Azure Container Registry client library
+
+```bash
+export DEBUG=azure*
+```
```

Comment:
We should update our template logging section to reference `AZURE_LOG_LEVEL` instead of `DEBUG`, maybe to match what `identity` has:
https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/identity/identity#logging

Reply:
Updated, will submit a separated PR for template.

Reply:
https://github.com/Azure/azure-sdk-for-js/pull/14022/files